source/Publish Profiles/Pre-production/phe-test-reporting.publish.xml

```diff
@@ -0,0 +1,112 @@
+    <DoNotDropLinkedServers>True</DoNotDropLinkedServers>
+    <DoNotDropLogins>True</DoNotDropLogins>
+    <DoNotDropMessageTypes>True</DoNotDropMessageTypes>
+    <DoNotDropPartitionFunctions>True</DoNotDropPartitionFunctions>
+    <DoNotDropPartitionSchemes>True</DoNotDropPartitionSchemes>
+    <DoNotDropPermissions>True</DoNotDropPermissions>
+    <DoNotDropQueues>True</DoNotDropQueues>
+    <DoNotDropRemoteServiceBindings>True</DoNotDropRemoteServiceBindings>
+    <DoNotDropRoleMembership>True</DoNotDropRoleMembership>
+    <DoNotDropRoutes>True</DoNotDropRoutes>
+    <DoNotDropRules>True</DoNotDropRules>
+    <DoNotDropScalarValuedFunctions>False</DoNotDropScalarValuedFunctions>
+    <DoNotDropSearchPropertyLists>True</DoNotDropSearchPropertyLists>
+    <DoNotDropSecurityPolicies>True</DoNotDropSecurityPolicies>
+    <DoNotDropSequences>True</DoNotDropSequences>
+    <DoNotDropServerAuditSpecifications>True</DoNotDropServerAuditSpecifications>
+    <DoNotDropServerRoleMembership>True</DoNotDropServerRoleMembership>
+    <DoNotDropServerRoles>True</DoNotDropServerRoles>
+    <DoNotDropServerTriggers>True</DoNotDropServerTriggers>
+    <DoNotDropSignatures>True</DoNotDropSignatures>
+    <DoNotDropServices>True</DoNotDropServices>
+    <DoNotDropStoredProcedures>False</DoNotDropStoredProcedures>
+    <DoNotDropSymmetricKeys>True</DoNotDropSymmetricKeys>
+    <DoNotDropSynonyms>True</DoNotDropSynonyms>
+    <DoNotDropTableValuedFunctions>False</DoNotDropTableValuedFunctions>
+    <DoNotDropTables>False</DoNotDropTables>
+    <DoNotDropUserDefinedDataTypes>False</DoNotDropUserDefinedDataTypes>
+    <DoNotDropUserDefinedTableTypes>False</DoNotDropUserDefinedTableTypes>
+    <DoNotDropUsers>True</DoNotDropUsers>
+    <DoNotDropViews>False</DoNotDropViews>
+    <DoNotDropXmlSchemaCollections>True</DoNotDropXmlSchemaCollections>
+    <DropExtendedPropertiesNotInSource>False</DropExtendedPropertiesNotInSource>
+  </PropertyGroup>
+  <ItemGroup>
+    <SqlCmdVariable Include="ETS">
+      <Value>NTBS_ETS_Test</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="Labbase2">
+      <Value>NTBS_Labbase2_Test</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="LTBR">
+      <Value>NTBS_LondonTBR_Test</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="migration">
+      <Value>NTBS_Migration_Test</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="NTBS">
+      <Value>NTBS_Test</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="NTBS_AUDIT">
+      <Value>NTBS_Audit_Test</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="NTBS_R1_Geography_Staging">
+      <Value>NTBS_R1_Geography_Staging_Test</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="NTBS_Specimen_Matching">
+      <Value>NTBS_Specimen_Matching_Test</Value>
+    </SqlCmdVariable>
+    <SqlCmdVariable Include="NTBS_URL">
+      <Value>https://ntbs-test.apps.ocp-por.unix.phe.gov.uk/</Value>
```

Comment:
Has this been confirmed?

Reply:
Not yet, so currently a placeholder of what it is expected to be